lib/cPanel/PublicAPI.pm

```diff
@@ -289,6 +289,17 @@ sub api_request {
             }
         }
 
+        if ($self->{'operating_mode'} eq 'accesshash') {
+            my $token_app = ($service eq 'whostmgr') ? 'whm' : $service;
```

Comment:
@FGasper  Shouldn't this be uppercase to maintain compat?

Reply:
Documentation gives it in lowercase: https://documentation.cpanel.net/display/DD/Guide+to+API+Authentication+-+API+Tokens+in+WHM

Reply:
Confirm lowercase works as well
 elsif ( Cpanel::App::is_whm() && $authtype =~ m{whm}i ) {